components/ai_chat/core/common/features.cc

```diff
@@ -60,4 +60,16 @@ bool IsPageContentRefineEnabled() {
   return base::FeatureList::IsEnabled(features::kPageContentRefine);
 }
 
+BASE_FEATURE(kOpenLeoFromBraveSearch,
+             "OpenLeoFromBraveSearch",
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
should this condition also include iOS? Also, why is it disabled on android? Nearly all the logic should be the same, aside from the UI-opening delegate.

Reply:
I think we could just enable the whole feature when it's done.
Currently this feature flag is mainly gating the navigation throttle, and without UI opening, there's really no much point have it.

Reply:
I'll add iOS, tho currently this feature flag is used only in webui and contents folder, so not really affecting anything on iOS atm.

Reply:
added ios in https://github.com/brave/brave-core/commit/ab4ea07f080379399d945e2357de36394ecf6fc0